src/main/java/com/googlecode/lanterna/gui2/MultiWindowTextGUI.java

```diff
@@ -46,12 +46,17 @@
     private final WindowManager windowManager;
     private final BasePane backgroundPane;
     private final List<Window> windows;
+    private final List<Window> stableOrderingOfWindows;
```

Comment:
I'm still not sure why you need this? The `window` list should contain all the windows ordered by Z-index.

Reply:
Let's focus on the other pull request and leave this one dormant for now.